apps/documentation/src/routes/documentation/tutorial/api-fetching.mdx

```diff
@@ -103,3 +107,13 @@ export default function IndexPage({
 ```
 
 Refresh the browser now! A bit ugly, but all the Pokémon are finally printed on screen!
+
+import NavigationButtons from '../../../components/navigation-buttons'
+
+<NavigationButtons
+  prev={{
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Don’t put a space after the opening parenthesis. (COMMA_PARENTHESIS_WHITESPACE)
Suggestions: `{`
Rule: https://community.languagetool.org/rule/show/COMMA_PARENTHESIS_WHITESPACE?lang=en-US
Category: TYPOGRAPHY
<!-- __reviewdog__:ChAzN2E4NWEyZjU0NmE1NWZjEgxMYW5ndWFnZVRvb2w= -->